.ci/compute_projects.py

```diff
@@ -52,6 +52,8 @@
     "clang": {"clang-tools-extra", "compiler-rt", "cross-project-tests"},
     "clang-tools-extra": {"libc"},
     "mlir": {"flang"},
+    # Test everyything if ci scripts are changed.
+    ".ci": {"llvm", "clang", "lld", "lldb"},
```

Comment:
This doesn't seem to test everything? Additional projects should get added on later due to dependency propagation, but I think this is a bit confusing in its current form. I would prefer to have everything explicitly enumerated, but a comment would also probably be fine.

Reply:
It doesn't seem like anything actually got fixed here?

Reply:
Sorry for missing the comment, LGTM.